### ISSUE TITLE
v1.8.1 on Mac : impossible to open file with password

### ISSUE THREAD

**Issue by micoci2** [moneymanagerex/moneymanagerex#6978]
MMEX version:
 - [ ] 1.7.0
 - [] 1.8.0
 - [X ] 1.8.1
 - [ ] 1.8.2
 - [ ] Other (please specify)

Note: bug reporters are expected to have verified the bug still exists
either in the last stable version of MMEX or on updated development code
(master branch).

Operating System:
 - [] Windows
 - [X ] Mac OSX
 - [ ] Linux 

# Description of the bug
I downloaded the new v1.8.1 and installed it on my Macbook Pro on macOS Monterey.
I "replaced" the previous version.
When launching the application, it asked the password of my encrypted file (normal behavior).
I typed the password and it did not recognized it. It did it several times and it was not possible to open it. I was sure of my password.
I installed back the v1.8.0. The file opened directly after typing the password.

# Reproduction

Is the bug reproducible? 
 - [x] Always 
 - [ ] Randomly 
 - [ ] Happened only once

Reproduction steps:
Install the v1.8.1 to replace a v1.8.0.
Open an encrypted file using its password.

Expected result:
The file should open after typing the password.

Actual result:
The v1.8.1 does not accept my password. The v1.8.0 does.

# Additional information

If you have a backtrace for a crash or a warning, paste it here.

**Comment by renato-mmex** [comment]
https://github.com/moneymanagerex/moneymanagerex/issues/6971

**Comment by warslet** [comment]
**I downloaded the new v1.8.1 and installed it on my Macbook Pro on macOS Monterey.
I "replaced" the previous version.
When launching the application, it asked the password of my encrypted file (normal behavior).
I typed the password and it did not recognized it. It did it several times and it was not possible to open it. I was sure of my password.
I installed back the v1.8.0. The file opened directly after typing the password.****
**__**
I have the same specific issue and have never had to re-install a **previous** version. I would appreciate it if I could be given a simple guide as to how to revert on my MacBook.
